src/Components/Web.JS/src/InputFile.ts

```diff
@@ -57,6 +57,7 @@ async function toImageFile(elem: InputElement, fileId: number, format: string, m
   const loadedImage = await new Promise(function(resolve: (loadedImage: HTMLImageElement) => void): void {
     const originalFileImage = new Image();
     originalFileImage.onload = function(): void {
+      URL.revokeObjectURL(originalFileImage.src);
```

Comment:
Should we also do this `onerror`? Is there an opportunity for that to happen?

Reply:
Sure, can't hurt. Note, we'll have to `onerror = null` first though: https://stackoverflow.com/questions/8124866/how-does-one-use-the-onerror-attribute-of-an-img-element

Reply:
https://github.com/dotnet/aspnetcore/pull/35499/commits/4bb17a6f6f7fb783dd3c30ac1f92f574da3f4b40

Reply:
Good idea. I guess it would error out if the file sent wasn't an image?

@TanayParikh - Looks like the compiled dist/Release/blazor.*.js files didn't get this last update. From my understanding those also has to be committed?

Reply:
> Good idea. I guess it would error out if the file sent wasn't an image?

More so if the image doesn't exist at all: 
https://www.w3schools.com/jsref/event_onerror.asp

> @TanayParikh - Looks like the compiled dist/Release/blazor.*.js files didn't get this last update. From my understanding those also has to be committed?

Usually yes, however given the final change wouldn't have any effect on tests, updating the file isn't strictly necessary here (I also didn't have push permissions via git due to this branch being on a separate fork, only could make changes via GitHub UI). Regardless, the update will be picked up automatically via the next change-set. Good eye though! 😄